.restyled.yaml

```diff
@@ -0,0 +1,69 @@
+enabled: true
+
+auto: false
+
+# Open Restyle PRs?
+pull_requests: true
+
+# Leave comments on the original PR linking to the Restyle PR?
+comments: true
+
+# Set commit statuses on the original PR?
+statuses:
+  # Red status in the case of differences found
+  differences: true
+  # Green status in the case of no differences found
+  no_differences: true
+  # Red status if we encounter errors restyling
+  error: true
+
+# Request review on the Restyle PR?
+#
+# Possible values:
+#
+#   author: From the author of the original PR
+#   owner: From the owner of the repository
+#   none: Don't
+#
+# One value will apply to both origin and forked PRs, but you can also specify
+# separate values.
+#
+#   request_review:
+#     origin: author
+#     forked: owner
+#
+request_review: author
+
+# Add labels to any created Restyle PRs
+#
+# These can be used to tell other automation to avoid our PRs.
+#
+labels:
+  - restyled
+  - 'Skip CI'
+
+# Labels to ignore
+#
+# PRs with any of these labels will be ignored by Restyled.
+#
+# ignore_labels:
+#   - restyled-ignore
+
+# Restylers to run, and how
+restylers:
+  - name: prettier
+    image: 'restyled/restyler-prettier:v3.3.2-2'
+    command:
+      - prettier-with-tailwindcss
```

Comment:
Did you mean `prettier`? I'm not sure what `prettier-with-tailwindcss` is

Reply:
I referenced the official documentation, and it appears that at some point they switched to using that binary instead.
Related: https://github.com/restyled-io/restylers/commit/6c1ddee30ce8e005428244f44c23c6b255485b31

Reply:
By the way, I misunderstood your comment, if you want to run it manually,
you can install that package with the following command:
```
yarn add --dev prettier prettier-plugin-tailwindcss
```

Reply:
So the `restyled/restyler-prettier` image is built with tailwindcss, but I don't think we care about that.

Can we try with just the command?

```
    command:
      - prettier
      - '--write'
```

The `prettier` binary is avaialble in the image

```
$ docker run -it restyled/restyler-prettier:v3.3.2-2 prettier -v
3.3.2
```

Reply:
Sure, do you want to change that?

Reply:
Go ahead and make that change

Reply:
Cool, I made changes.

Reply:
Thanks for the review, approval. :)
Can I merge it? Or do I have to wait for another review?